ui/postcss.config.js

```diff
@@ -1,5 +0,0 @@
-module.exports = {
-  plugins: [
-    require('autoprefixer')
-  ]
-};
```

Comment:
- [ ] We do not need it becasue of `browserslist` in `ui/package.json`?

Reply:
I belive so.
https://facebook.github.io/create-react-app/docs/post-processing-css